### DISCUSSION TITLE
Album properties can be changed with PHOTOPRISM_READONLY=true

### DISCUSSION THREAD

**Discussion by ruudhop** [photoprism/photoprism#2511] [Q&A]
Hi all,

New photoprism user here :-)
I've installed photoprism using the docker compose method on Debian 11. After uploading photos to an album I want to change photoprism to readonly and public.
So I've changed the docker-compose.yml file:
PHOTOPRISM_PUBLIC:  "true"
PHOTOPRISM_READONLY:  "true"
PHOTOPRISM_DISABLE_SETTINGS:  "true"

After restarting photoprism no password is needed and all upload and edit functionality for photos is gone (as it should be), but I can still edit the properties of an **album**! So anyone can change the name/location/etcetera of an album.

Did I miss something in the config or is this a bug?

Regards,
Ruud

**Comment by centralhardware** [comment]
This property make original folder read only not photoprism data. I think so. If this property work as you write face recognition also must don't work in this mode.

**Reply by ruudhop** [reply]
Well, can't check that right now, because face recognition is also disabled via
PHOTOPRISM_DISABLE_FACES: "true"
(I don't need face recognition)

**Reply by lastzero** [reply]
Read only refers to the originals folder. Not the UI.

**Comment by ruudhop** [comment]
So PHOTOPRISM_READONLY: "true" makes the folder with images readonly. And by doing this a lot of functionality is blocked, but not editing Album properties. 
So I thought: let's disable "Edit" in the UI settings. But that has the same result: editing image properties is blocked, but Album properties are still editable.

Anyone know how I can disable editing Album properties?

**Reply by graciousgrey** [reply]
That's not possible at the moment. 
Please note that our public mode has been built for our public demo and to use PhotoPrism without authentication in a local network.

A public image (#669) wall as well as multi-user support (#98) including read-only users are planned

**Reply by ruudhop** [reply]
Ok thanks, I'll be patiently waiting for a new version :-)
